### ISSUE TITLE
Tabbing to select buttons for accessibility is really hard to see

### ISSUE THREAD

**Issue by srietkerk** [microsoft/pxt-microbit#5012]
When using tab to select the buttons in the tour, you can't really see the outline of the box at all. We should have the tab boxes be more pronounced for those who want to use keyboard controls.

**To Reproduce**
1. Start the tour
2. Try tabbing to select the button

**When the back is highlighed** 
![image](https://user-images.githubusercontent.com/49178322/227595332-4c26cf69-9782-476c-b8aa-32e72b41fceb.png)

**When the "x" is highlighed** 
![image](https://user-images.githubusercontent.com/49178322/227595356-48ed8556-49a8-41a9-83ea-09446ea9e73f.png)

**When next is highlighted**
<img width="73" alt="image" src="https://user-images.githubusercontent.com/49178322/227595307-c8cdd393-4699-45b7-95c4-9ca122ea27f1.png">

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Chrome

**Comment by kimprice** [comment]
@srietkerk Is this better? Do you have a suggestion for the design? I think the following looks cleaner and you can see the difference when tabbing, but it's hard to tell what is selected when just stationary.
![tabButton](https://user-images.githubusercontent.com/15070078/229173419-f636a694-be9a-4853-87c7-84f2b77abc54.gif)

**Comment by kimprice** [comment]
I could make the border larger:
![tabButton2](https://user-images.githubusercontent.com/15070078/229175052-26337cac-97fe-414c-a9eb-f83785d74611.gif)
